parser.js

```diff
@@ -903,6 +903,62 @@ class DataflashParser {
         this.parseAtOffset(type)
         console.log('done')
     }
+
+    // Return array of objects giving stats about the composition of the log, sizes in bytes
+    stats() {
+
+        // Need this to access get_size_of in the helper function
+        const get_size_of_fun = this.get_size_of
+        function get_msg_size(Format) {
+            // Work out size of single message, all message have a 3 byte header
+            let size = 3
+            for (let i = 0; i < Format.length; i++) {
+                size += get_size_of_fun(Format.charAt(i))
+            }
+            return size
+        }
```

Comment:
```suggestion
        get_msg_size(Format) => {
            // Work out size of single message, all message have a 3 byte header
            let size = 3
            for (let i = 0; i < Format.length; i++) {
                size += this.get_size_of(Format.charAt(i))
            }
            return size
        }
```

Arrow functions automatically bind to `this` from the outer scope

Reply:
Thanks, done.